bin/crunchy-exporter/exporter_uid_daemon.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
Can we use the existing `uid_daemon.sh` that is already under `bin`?

Reply:
Sure, on review, the differences between the files seem to be inconsequential for this implementation. Updated, thanks.